src/modules/72-templates-library/components/DeleteTemplateModal/DeleteTemplateModal.tsx

```diff
@@ -96,7 +99,12 @@ export const DeleteTemplateModal = (props: DeleteTemplateProps) => {
     }
   }, [templatesError])
 
-  const performDelete = async (commitMsg: string, versions?: string[]): Promise<void> => {
+  const [commitMessage, setCommitMessage] = React.useState<string>(`${getString('delete')} ${template.name}`)
+  React.useEffect(() => {
+    setCommitMessage(`${getString('delete')} ${template.name}`)
+  }, [getString, template])
```

Comment:
state and effect are not needed here

```suggestion
  const commitMessage = `${getString('delete')} ${template.name}`
```

Reply:
changed.